designer/client/lists-edit.js

```diff
@@ -12,27 +12,29 @@ class ListsEdit extends React.Component {
     })
   }
 
-  onClickAddList = (e, list) => {
+  onClickAddList = async e => {
     e.preventDefault()
-
+    const { data } = this.props
+    const id = await data.getId()
     this.setState({
+      id: id,
```

Comment:
just { id, showAddList: true } should work

Reply:
Fixed